README.rst

```diff
@@ -202,6 +202,29 @@ depends on JavaScript which uses solely double-precision floats both for integer
 which can lead to unexpected truncation.
 
 
+Style Check
+-------------------------
+To check the compliance of a Swagger specification at ``/some/path/swagger.yaml`` to the Swagger style guides, invoke:
```

Comment:
I'd add an introductory sentence here. For example:
We found it important to have a uniform documentation style across all the Swagger specifications in an organization. To that end, swagger_to includes an utility to automatically check the style such as casing of the definition names, property names, descriptions and verb moods (present tense instead of imperative).